dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsConnector.java

```diff
@@ -30,19 +31,39 @@
 @ParametersAreNonnullByDefault
 public class OracleStatsConnector extends AbstractOracleConnector {
 
+  static int MAX_DAYS = 10000;
+
   public OracleStatsConnector() {
     super(OracleConnectorScope.STATS);
   }
 
   @Override
   public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments) throws Exception {
     StatsTaskListGenerator taskListGenerator = new StatsTaskListGenerator();
-    out.addAll(taskListGenerator.createTasks(arguments));
+    int queriedDays = getQueryLogDays(arguments);
+    out.addAll(taskListGenerator.createTasks(arguments, queriedDays));
   }
 
   @Nonnull
   @Override
   public String summary(String fileName) {
     return String.format("Oracle statistics saved to %s", fileName);
   }
+
+  static int getQueryLogDays(ConnectorArguments arguments) {
```

Comment:
I would suggest using `Duration` as the return type.

Reply:
Done